e2e/tests/functional/search.e2e.spec.js

```diff
@@ -139,68 +137,86 @@ test.describe("Search Tests @unstable", () => {
         await expect(searchResults).toHaveText(folderName);
     });
 
+    test('Search results are debounced @couchdb', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6179'
+        });
+        await createObjectsForSearch(page);
+
+        let networkRequests = [];
+        page.on('request', (request) => {
+            const searchRequest = request.url().endsWith('_find');
+            const fetchRequest = request.resourceType() === 'fetch';
+            if (searchRequest && fetchRequest) {
+                networkRequests.push(request);
+            }
+        });
+
+        // Full search for object
+        await page.type("input[type=search]", 'Clock', { delay: 100 });
+
+        // Wait for search to finish
+        await waitForSearchCompletion(page);
+
+        // Network requests for the composite telemetry with multiple items should be:
+        // 1.  batched request for latest telemetry using the bulk API
+        expect(networkRequests.length).toBe(1);
+
+        const searchResultDropDown = await page.locator(searchResultDropDownSelector);
+
+        await expect(searchResultDropDown).toHaveText('Clock A');
```

Comment:
This is looking for an exact match and fails, let's update the expect to use a regex:

```js
await expect(searchResultDropDown).toHaveText(/Clock A/);
```

Reply:
ah, meant this to be `toContainText` as above. nice catch